docs/operator-bundle/testing-and-releasing.md

```diff
@@ -0,0 +1,181 @@
+# Testing and releasing the operatorhub bundle for ASO
+
+This is a guide to producing and testing the operator bundle after an ASO release.
```

Comment:
Suggest giving a definition for ASO just in case people are unsure when reading it.

```suggestion
This is a guide to producing and testing the operator bundle after an Azure Service Operator (ASO) release.
```

Reply:
Updated, along with a note on intended audience. (It's us. :wink:)